azkaban-common/src/main/java/azkaban/user/Permission.java

```diff
@@ -191,6 +191,7 @@ public int toFlags() {
     SCHEDULE(0x0000008),
     METRICS(0x0000010),
     CREATEPROJECTS(0x40000000), // Only used for roles
+    UPLOADPROJECTS(0x0008000),
```

Comment:
Immediately, I don't plan to make use of the finer grained control. But, I do foresee a case where we don't want to make everyone admin to be able to upload projects.